Documentation/git-config.txt

```diff
@@ -253,14 +257,6 @@ $XDG_CONFIG_HOME/git/config::
 $GIT_DIR/config::
 	Repository specific configuration file.
 
-On Windows, as there is no central `/etc/` directory, there is yet another
-config file (located at `$PROGRAMDATA/Git/config`), intended to contain
-settings for *all* Git-related software running on the machine. Consequently,
-this config file takes an even lower precedence than the
-`$(prefix)/etc/gitconfig` file. Typically `$PROGRAMDATA` points to
-`C:\ProgramData` (on Windows XP the equivalent in `$ALLUSERSPROFILE` is used,
-i.e. `C:\Documents and Settings\All Users\Application Data\Git\config`).
-
```

Comment:
The upstream text puts all details about individual files into the list above. The XDG user-config and the standard ~/.gitconfig demonstrate this quite well. After the list only generic stuff about read / write priorities are described. Putting one block with in-depth information about one file between the list and the text is exactly the thing I'd like to avoid. So if the text must be kept, I'd put it into the appropriate list item.